config/web/web.go

```diff
@@ -21,6 +22,21 @@ type Config struct {
 
 	// Port to listen on (default to 8080 specified by DefaultPort)
 	Port int `yaml:"port"`
+
+	// TLS configuration
+	Tls TlsConfig `yaml:"tls"`
+
+	tlsConfig      *tls.Config
+	tlsConfigError error
```

Comment:
Why are we persisting the error in the struct?

Reply:
See below